sdk/nodejs/tests/provider/experimental/testdata/component-description/index.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright 2025-2025, Pulumi Corporation.  All rights reserved.
+
+import * as pulumi from "@pulumi/pulumi";
+
+// biome-ignore lint: MyComponentArgs needs to be an interface but biome doesn't like that
+export interface MyComponentArgs {}
```

Comment:
Will input/output/type descriptions come in a later PR?

Reply:
Yes, I'm still working on figuring those out.